cmd/bnsd/app/lateinit.go

```diff
@@ -0,0 +1,22 @@
+package bnsd
+
+/*
+import (
+	"github.com/iov-one/weave"
+	"github.com/iov-one/weave/x/lateinit"
+	"github.com/iov-one/weave/x/msgfee"
+)
+
+func init() {
+	// cond:gov/rule/0000000000000003 is owned by technical executors
```

Comment:
If I understand correctly, the `cond:gov/rule/0000000000000002` (economic executors) should set the product fees and thus should be admin. @davepuchyr?

The owner however could be the technical executors